src/postBackEnd.js

```diff
@@ -0,0 +1,173 @@
+import * as firebase from 'firebase';
+import { db } from './index.js';
+import { currentUserObj, userExist } from './signIn.js';
+
+
+///////////////////////////////////////////////////////
+//Post gets and sets
+///////////////////////////////////////////////////////
+
+export function getAddress(post_id){
+  const collect = db.collection('post')//get wanted collection
+  query = collect.where('post_id', '==', post_id)
+  query.get().then(queriedDocs => {
+      if (queriedDocs.empty == false) {
+          queriedDocs.forEach(singleDoc => {
+              return singleDoc.data().address;
+          })
+      } else {
+          console.log("No docs match");
+      }
+  });
+}
+
+export function setAddress(post_id, newAddress){
+  db.collection('post').doc(post_doc).update({
+      addres: newAddress
+  }).catch(function (error) {
+      // The document probably doesn't exist.
+      console.error("Error updating document: ", error);
+      return false
+  });
+  return true
+}
+
+// Thinking it might be better to simply do one search for every comment
+// item with a given post_id than a serach for a specific comment id for
+// every value in an array of comments.  Will talk with Branden about this
+
+// export function getComments(post_id){

```

Comment:
Nathan I didn't realize you wrote code for getting comments I just pushed up my own version with test a couple of days ago.  We should talk about merging our two versions.

Reply:
The gets were just something I threw together for me to use in my random tests.  I tested some but no tests files are made, so if your gets work and you have a test file made then we can use those no problem.